### ISSUE TITLE
PayPal doesn't support localhost as return_uri. Add a flag?

### ISSUE THREAD

**Issue by sfxworks** [int128/kubelogin#262]
Oddly enough, they support http://127.0.0.1:8000 and that IP mapped to a subdomain of mine, just not the name _localhost_  😐. 

After contacting their tech support over a few days, they confirmed this. 

Can a flag be added to change the redirect_uri?

**Comment by int128** [comment]
Not supported at this time. I will add this feature.

**Comment by int128** [comment]
I just released v1.19.0. Would you check it?

**Comment by sfxworks** [comment]
Works well! Just, seems to be another issue I've stumbled upon. 

```
error: get-token: could not get a token: authentication error: authentication error: authorization code flow error: id_token is missing in the token response: &{A23AAFRE6LMrVcGGs-R0-Q2_d4OXSPSxyttBEScpyfHzYATxNNUp5LbGongehMWE2IEVdLmdy_9ITKd5LCtR4V9CqBtgKT9kQ Bearer R23AAG8IETOvFcXV5M4x6RM_eJss7C_t7qS5wCChhH5js0qkzHDW1UjlPrz5J52zwleSWQ16uFkqqRvoZDynObpS3jZ-_05vboyR5oL29mFbI7UVq7jmmT21dJ5gcwJwHNIZohY3bRwULOZdbfofg 2020-04-13 03:32:23.441696 -0400 EDT m=+28807.177359801 map[access_token:A23AAFRE6LMrVcGGs-R0-Q2_d4OXSPSxyttBEScpyfHzYATxNNUp5LbGongehMWE2IEVdLmdy_9ITKd5LCtR4V9CqBtgKT9kQ expires_in:%!s(float64=28800) nonce:2020-04-12T23:32:21ZFu2iqLKCDO9DEPzwMKc0fC4eMsBu1IVDdazWXFPN-Lk refresh_token:R23AAG8IETOvFcXV5M4x6RM_eJss7C_t7qS5wCChhH5js0qkzHDW1UjlPrz5J52zwleSWQ16uFkqqRvoZDynObpS3jZ-_05vboyR5oL29mFbI7UVq7jmmT21dJ5gcwJwHNIZohY3bRwULOZdbfofg scope:https://uri.paypal.com/services/identity/credentialmanagement/unbind https://uri.paypal.com/services/identity/credentialmanagement https://uri.paypal.com/services/identity/proxyclient openid https://api.paypal.com/v1/vault/credit-card/.* https://uri.paypal.com/services/subscriptions https://uri.paypal.com/services/applications/webhooks email https://uri.paypal.com/services/identity/grantdelegation https://api.paypal.com/v1/vault/credit-card token_type:Bearer]}
```

Requesting more scope issues and going from there. I'm guessing I need the `profile` scope to get it properly based on some of their documentation, marketing this new issue as unrelated or a new issue entirely.
